modules/security-context-constraints-psa-opting.adoc

```diff
@@ -10,18 +10,7 @@ You can enable or disable automatic pod security admission synchronization for m
 
 [IMPORTANT]
 ====
-Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
-
-* `default`
-* `kube-node-lease`
-* `kube-system`
-* `kube-public`
-* `openshift`
-* All system-created namespaces that are prefixed with `openshift-`, except for `openshift-operators`
-
-By default, all namespaces that have an `openshift-` prefix are not synchronized. You can enable synchronization for any user-created [x-]`openshift-*` namespaces. You cannot enable synchronization for any system-created [x-]`openshift-*` namespaces, except for `openshift-operators`. 
-
-If an Operator is installed in a user-created `openshift-*` namespace, synchronization is turned on by default after a cluster service version (CSV) is created in the namespace. The synchronized label inherits the permissions of the service accounts in the namespace.
+You cannot enable pod security admission synchronization on some system-created namespaces. For more information, see _Pod security admission synchronization exclusions_.
```

Comment:
@bergerhoffer Is the link for "Pod security admission synchronization exclusions." missing here or it is supposed to be just a text?

Reply:
@gkarager It is supposed to be just text.

Unfortunately, we can't put cross-references to other areas of our doc set in modules (anything in the modules/ directory). If we need to link, we usually add a link right after the module is included, in an additional resources section.

Doesn't look like I did that this time, maybe because it's so close to it above. But it looks like I changed the name in the section, but didn't here, so that's not helpful here. I will add a link right after the module here.